test/parallel/test-fs-chmod.js

```diff
@@ -139,5 +153,7 @@ if (fs.lchmod) {
 
 
 process.on('exit', function() {
+  fs.chmodSync(file1, mode_sync);
```

Comment:
OK, well in that case, either surrounding it with `if (common.isWindows)` and/or providing a comment explaining why it's there might help prevent someone like me from coming along and obliviously removing it in six months. :-D